subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/versions/KotlinGradlePluginVersions.groovy

```diff
@@ -29,7 +33,25 @@ class KotlinGradlePluginVersions {
         '1.6.0', '1.6.10',
     ]
 
+    private static final VersionNumber KOTLIN_GRADLE_PLUGIN_1_5_0_VERSION_NUMBER = VersionNumber.parse('1.5.0')
```

Comment:
I wonder if this constant should more describe the intent, something like `KOTLIN_VERSION_SUPPORTING_JAVA_17`, WDYT?

Reply:
Removed, since I will use `-add-opens` workaround